tools/flecstan/CMakeLists.txt

```diff
@@ -0,0 +1,36 @@
+
+# CMake
+project(flecstan LANGUAGES CXX)
+cmake_minimum_required(VERSION 3.8) # probably not necessary
+
+# C++
+set(CMAKE_CXX_COMPILER g++)
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++17")
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -pthread -fno-rtti")
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -pedantic -Wall -s")
+
+# Header file location
+include_directories(src)
+
+# Source files
+file(GLOB SOURCES "src/*.cc")
+
+# Executable
+###set(CMAKE_RUNTIME_OUTPUT_DIRECTORY "/usr/local/bin") 
+add_executable(flecstan ${SOURCES})
+
+# Libraries
+target_link_libraries(
+   # executable...
+   flecstan
+   # libraries...
+   clangAnalysis clangAST clangBasic clangDriver clangEdit clangFrontend
```

Comment:
Hard-codes libraries are a big no-no, use `find_package(LLVM)` (from `cinch`) and then `${LLVM_LIBRARIES}`.

Reply:
Some distros call the libraries `libLLVM*-7.so`.